server/cluster.go

```diff
@@ -557,6 +557,13 @@ func (c *RaftCluster) checkStores() {
 	}
 }
 
+// GetStores gets stores from cluster.
```

Comment:
It seems this comment is not for this function?

Reply:
Yeah, :grin: